examples/direct-integration/src/App.js

```
@@ -55,7 +54,6 @@ export default function Home() {
       }
     } catch (error) {
       console.error(error)
-      setMessage(error.message);
```

do you want to hide the error from the user?